flixpy/flix/lib/types.py

```diff
@@ -2374,6 +2382,7 @@ def from_dict(
         into.shot_id = data.get("id")
         into.show_id = data.get("show_id")
         into.sequence_id = data.get("sequence_id")
+        into.origin = data.get("origin")
```

Comment:
```suggestion
        into.origin = data.get("origin", "")
```
`data.get(...)` will have the type `T | None` (e.g. `str | None` if it's a `str` field) but `origin` is a `str`, i.e. it needs to always be set to a valid string, so we should provide a default value if the origin for some reason isn't set